### PR TITLE
[US 30 Task 102] added issues in the api

### PR DESCRIPTION
**Pull request by sverma89asu**

### CODE REVIEW

**Review by achaur11 — APPROVED** [review]

**Comment by spark290** [comment]
LGTM

File: taigaProject/src/taigaApi/controller/metric_controller.py [hunk 1]
```diff
@@ -37,4 +39,9 @@ def get_burndown_chart_metric(burndownChartRequest: BurndownChartRequest, token:
 
 @router.post("/metric/Cruft")
 def get_zero_business_value(cruftRequest: CruftRequest, token: Annotated[str | None, Header()] = None):
-    return get_zero_business_value_user_stories(cruftRequest.projectId, cruftRequest.startDate, cruftRequest.endDate, cruftRequest.attributeKey, token)
+    zero_bv_stories = get_zero_business_value_user_stories(cruftRequest.projectId, cruftRequest.startDate, cruftRequest.endDate, cruftRequest.attributeKey, token)
+    issues = get_issues(cruftRequest.projectId, date.fromisoformat(cruftRequest.startDate), date.fromisoformat(cruftRequest.endDate), token)
+    return {
+    "zero_bv_us": zero_bv_stories,
```
Comment (shahih25) [inline taigaProject/src/taigaApi/controller/metric_controller.py]: nit: format

**Comment by spark290** [comment]
LGTM

**Review by shahih25 — APPROVED** [review]
LGTM

**Comment by spark290** [comment]
LGTM

**Pull request merged**